cmake/configure.cmake

```diff
@@ -60,7 +60,7 @@ else()
         message(FATAL_ERROR "Paddle need cudnn to compile")
     endif()
 
-    set(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS} "-Xcompiler ${SIMD_FLAG}")
+    set(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS} "-Xcompiler ${SIMD_FLAG}" "--expt-relaxed-constexpr")
```

Comment:
please add "--expt-relaxed-constexpr" in https://github.com/wangkuiyi/Paddle/blob/f6f312418da48cbd486ce49c4a4a778ab94d41b4/cmake/flags.cmake#L197, because it only works on CUDA >= 8.0